router-sharding/02_route-django-psql-example.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  labels:
+    app: django-psql-example
+    template: django-psql-example
+    type: internal
+  name: django-psql-example
+  namespace: test-sharding
+spec:
+  host: django-psql-example-test-sharding.internalapps.top.sandbox37.opentlc.com
```

Comment:
The host should be removed so that it defaults to whatever the cluster subdomain is.